packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -37,7 +37,7 @@ export function ensureInitialized(...expectedFrameContexts: string[]): void {
  *
  * @internal
  */
-export function isAPISupportedByPlatform(requiredVersion: string = defaultSDKVersionForCompatCheck): boolean {
+export function isCurrentSDKVersionAtLeast(requiredVersion: string = defaultSDKVersionForCompatCheck): boolean {
```

Comment:
Not sure about this naming convention but not sure what else to name it

Reply:
what would happen if someone calls `isCurrentSDKVersionAtLeast`  with an older version of the SDK passed into `requiredVersion`, I think that would break the naming convention here

Reply:
these are internal functions, so this shouldn't break anything, but I am new to the backwards  compat stuff so i could be wrong

Reply:
Oh I didn't mean actually breaking anything, I meant that the name wouldn't make sense in that situation

Reply:
I think this name makes sense doesn't it? If GlobalVars.clientSupportedSDKVersion == 2.0 and requiredVersion == 1.0, the function would return true because the current version was at least 1.0. I suppose it's just a different way of saying "GreaterThanOrEqualTo".

I guess that's my way of saying "I think this name makes sense but if people want to pick an even clearer name it's okay with me".

Reply:
i explained this to @jadahiya-MSFT in teams and not here, I think if you know the parameters being passed in are '2.0' and not 'some random string', it totally makes sense.